lib/SyTest/Federation/Client.pm

```diff
@@ -248,14 +248,11 @@ sub join_room
       $self->do_request_json(
          method   => "PUT",
          hostname => $server_name,
-         uri      => "/v1/send_join/$room_id/$event_id",
+         uri      => "/v2/send_join/$room_id/$event_id",
```

Comment:
I've got a feeling we were deliberately leaving this for now, since (a) /v2/send_join isn't specced yet; (b) compatibility with dendrite.